src/unit_tests/wrappers/wazuh/syscheckd/fim_db_wrappers.c

```diff
@@ -190,3 +192,33 @@ int __wrap_fim_db_sync_path_range(fdb_t *fim_sql,
 
     return mock();
 }
+
+/* Auxiliar functions */
+/**
+ * @brief This function load the expect and will_return calls for the wrapper of fim_db_get_path_range
```

Comment:
This function **loads**. Also, this doxygen comment should be in the _.h_ file instead.